src/main/java/atmosfears/AtmosFearsBE/database/SensorCode.java

```diff
@@ -0,0 +1,23 @@
+package atmosfears.AtmosFearsBE.database;
+
+public enum SensorCode {
+    MpKrakAlKras("al. Krasińskiego"),
```

Comment:
I would suggest moving this data to a JSON file. It will be easier to update without the necessity to look for mappings in code. Enums are usually used for data that doesn't change over time. In this case, much better is a hashmap with data loaded from a file.

Reply:
You mean changing the type of the SensorCode in the data model to a string and using that as a key to a hashmap? I think that could require data migration, and i'm not sure thats feasible right now

Reply:
I'll just keep it as is for now, I suppose - while you're correct, that this is something that shouldn't be hardcoded, I don't think this is that big of a deal, at least at this moment. Should there ever be a need to expand the list, this change can be implemented alongside it.